.github/workflows/ci.yml

```diff
@@ -118,8 +118,28 @@ jobs:
       - name: Build
         run: dotnet build --configuration=Release -p:TreatWarningsAsErrors=true Src
 
+  # Check that building the ILGPU samples for all target frameworks works.
+  build-samples:
```

Comment:
this could be written as a matrix since its using same `if` statement as `build-library`, so something like:

```    
    strategy:
        matrix:
            path:
            - Src
            - Samples
...
      - name: Build
        run: dotnet build --configuration=Release -p:TreatWarningsAsErrors=true ${{ matrix.path }}
```

Reply:
Thanks @76creates, that's a great idea.